client/src/components/PostedAlertsTablePage.js

```diff
@@ -0,0 +1,136 @@
+                                    },
+                                    {
+                                        title: "Alert ID",
+                                        field: "alert_id",
+                                    },
+                                    {
+                                        title: "FlightAware Flight ID",
+                                        field: "fa_flight_id",
+                                    },
+                                    {
+                                        title: "Ident",
+                                        field: "ident",
+                                    },
+                                    {
+                                        title: "Registration",
+                                        field: "registration",
+                                    },
+                                    {
+                                        title: "Aircraft Type",
+                                        field: "aircraft_type",
+                                    },
+                                    {
+                                        title: "Origin",
+                                        field: "origin",
+                                    },
+                                    {
+                                        title: "Destination",
+                                        field: "destination",
+                                    },
+                                ]}
+                                data={data.map(alert => (
+                                    {
+                                        id: alert.id,
+                                        time_alert_received: alert.time_alert_received,
+                                        long_description: alert.long_description,
+                                        short_description: alert.short_description,
+                                        summary: alert.summary,
+                                        event_code: alert.event_code,
+                                        alert_id: alert.alert_id,
+                                        fa_flight_id: alert.fa_flight_id,
+                                        ident: alert.ident,
+                                        registration: alert.registration,
+                                        aircraft_type: alert.aircraft_type,
+                                        origin: alert.origin,
+                                        destination: alert.destination,
+                                    }
+                                ))}
+                            />
+                            :
+                            <div className="posted-alerts-table-spinner">
+                                <Spinner animation="border" variant="primary"/>
+                            </div>
+                        }
+                    </div>
+                </div>
+            </div>
+        )
+    }
+}
```

Comment:
Blame Git for making me notice the inconsistency, can you please add a trailing newline to the files inside of `components`? This is to match the style of the scaffolding from App and index in the source root.
```suggestion
}

```

Reply:
Woops yeah I keep forgetting about that, bad habit of mine - shouldn't happen again